src/cmds/scripts/pbs_habitat.in

```diff
@@ -721,9 +722,14 @@ if [ -z "$PBS_HOME" ]; then
 	exit 1
 fi
 if [ ! -d "$PBS_HOME" ]; then
-	echo "*** $PBS_HOME directory does not exist"
-	echo "*** Run pbs_postinstall to create it"
-	exit 1
+	if [ -x "$PBS_EXEC/sbin/pbs_server" ]; then
+		component="server"
+	elif [ -x "$PBS_EXEC/sbin/pbs_mom" ]; then
+		component="execution"
+	else
+		echo "*** PBS will not create home in client installation"
+	fi
+	${PBS_EXEC}/libexec/pbs_postinstall $component $PBS_VERSION $PBS_EXEC $PBS_HOME "" "sameconf"
```

Comment:
I would much rather see us issue an error message indicating that the admin needs to run a command like:
WARNING: PBS_HOME not found in $PBS_HOME
Please update your configuration file or run the following command before starting PBS Pro:
${PBS_EXEC}/libexec/pbs_postinstall update